### DISCUSSION TITLE
Project dead?

### DISCUSSION THREAD

**Discussion by midnightsmith89** [feelfreelinux/octo4a#143] [Q&A]
No commits or updates, responses, or anything from the Dev since early November.  I love this project, I'd hate to see it die, and I'm not a coder...

**Comment by feelfreelinux** [comment]
Have been a little busy with another project. Next week I'm going to have quite a bit of time off, will try to solve the issues then :)

**Reply by feelfreelinux** [reply]
I will also try to prepare documentation for other developers, if anyone wants to help out

**Reply by supreme-helix** [reply]
Yes please. Some dev documentation would be great. Will help a lot

**Reply by Meguinness** [reply]
Can you please look in to  solve the app crashing when starting the camera server on Android TV boxes . Thank you

**Comment by midnightsmith89** [comment]
Hey thanks man! I donated cuz I really like this project, so useful for
those old phones and whatnot!
